api-tests/features/dl-auth-source-checks/dl-cri-dropout.feature

```diff
@@ -0,0 +1,309 @@
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'drivingLicence' CRI response
+    When I call the CRI stub with attributes and get an 'access_denied' OAuth error
+      | Attribute | Values          |
+      | context   | "check_details" |
+    Then I get a 'uk-driving-licence-details-not-correct' page response
+    When I submit an 'end' event
+    Then I get a 'prove-identity-another-way' page response
+    When I submit a 'postOffice' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response with context '<context>'
+    When I submit a 'next' event
+    Then I get a 'claimedIdentity' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'f2f' CRI response
+    When I submit 'kenneth-passport-valid' details with attributes to the async CRI stub
+      | Attribute          | Values                     |
+      | evidence_requested | <evidence-requested-value> |
+    Then I get a 'page-face-to-face-handoff' page response
+
+    # Return journey
+    When I start a new '<journey-type>' journey and return to a 'page-ipv-reuse' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a '<expected-identity>' identity without a 'dcmaw' VC
+
+    Examples:
+      | journey-type      | context    | evidence-requested-value                    | expected-identity |
+      | low-confidence    | lastChoice | {"scoringPolicy":"gpg45","strengthScore":2} | P1                |
+      | medium-confidence | null       | {"scoringPolicy":"gpg45","strengthScore":3} | P2                |
+
+  Scenario Outline: User backs out of DL CRI and selects to return to the RP - should not have a DCMAW VC
+    When I start a new '<journey-type>' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'drivingLicence' CRI response
+    When I call the CRI stub with attributes and get an 'access_denied' OAuth error
+      | Attribute | Values          |
+      | context   | "check_details" |
+    Then I get a 'uk-driving-licence-details-not-correct' page response
+    When I submit an 'end' event
+    Then I get a 'prove-identity-another-way' page response
+    When I submit a 'returnToRp' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P0' identity without a 'dcmaw' VC
```

Comment:
(We don't actually care about the DCMAW VC if they return with a P0 - they don't have an identity either way, but it doesn't hurt to check!)